lib/client.go

```diff
@@ -130,12 +188,6 @@ func (c *Client) CreateInterface() error {
 			},
 		},
 	})
-	if err != nil {
-		netlink.LinkDel(link)
-		return fmt.Errorf("error configuring vprox interface: %v", err)
```

Comment:
error configuring vprox interface

Reply:
actually, I'm going to move this message to wrap the error returned at the call site on line 80, since I think it's there where more context is needed

Reply:
Thanks!